sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/GenerateUnsafeRowJoiner.scala

```diff
@@ -88,8 +92,14 @@ object GenerateUnsafeRowJoiner extends CodeGenerator[(StructType, StructType), U
           s"$getLong(obj2, offset2 + ${(i - bitset1Words) * 8})"
         }
       }
-      s"$putLong(buf, ${offset + i * 8}, $bits);"
-    }.mkString("\n")
+      s"$putLong(buf, ${offset + i * 8}, $bits);\n"
+    }
+
+    val functions = mutable.ArrayBuffer.empty[String]
+    val args = "java.lang.Object obj1, long offset1, java.lang.Object obj2, long offset2"
+    val copyBitsets = ctx.splitExpressions(copyBitset, "copyBitsetFunc",
+      ("java.lang.Object", "obj1") :: ("long", "offset1") ::
+      ("java.lang.Object", "obj2") :: ("long", "offset2") :: Nil)
```

Comment:
Could you fix use the named arguments when calling `splitExpressions`, like what I did in https://github.com/apache/spark/pull/19790?